Sage/carbon
# fix(progress-tracker): add deprecation warnings for orientation and direction props

### Proposed behaviour


Adds deprecation warning for `orientation` and `direction` props

### Current behaviour


No deprecation warning for `orientation` and `direction` props

### Checklist




#### QA


### Additional context



### Testing instructions


Confirm only one warning displays in storybook (docs tab)

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
